Libraries/ConfigParser/inc/structs.h

```diff
@@ -0,0 +1,118 @@
+#pragma once
+
+#include <stdint.h>
+#include <string>
+
+namespace StreamingToolkit
+{
+	/* 
+	 * Server app configuration
+	 */
+	typedef struct
```

Comment:
i was going to ask why this instead of `struct StructName {}` and it led me to https://stackoverflow.com/questions/1675351/typedef-struct-vs-struct-definitions which is interesting and now i am okay with this style 😸

Reply:
It's a good read!